Resources/Textures/DeltaV/Objects/Decoration/Flora/flora_palmtree.rsi/meta.json

```
@@ -0,0 +1,23 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Leonardo DaBepis#4869 on discord / @leonardo-dabepis on Tumblr",
+    "size": {
+        "x": 96,
+        "y": 96
+    },
+    "states": [
+        {
+            "name": "palm01"
+        },
+        {
+            "name": "palm02"
+        },
+		{
+            "name": "palm03"
+        },
+        {
+            "name": "palm04"
+        }
+		]
+}
```

Missing newline at the end of the file
```suggestion
}

```